### PR TITLE
Add a Reverse Helper

### PR DESCRIPTION
**Pull request by spencer516**
For: https://github.com/DockYard/ember-composable-helpers/issues/19

### CODE REVIEW

File: tests/integration/helpers/reverse-test.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  this.render(hbs`
+    {{~#each (reverse array) as |item|~}}
+      {{~item~}}
+    {{~/each~}}
+  `);
+
+  assert.equal(this.$().text().trim(), 'bazbarfoo', 'array is reversed');
+});
+
+test('It does mutate the original array', function(assert) {
```
Comment (poteto) [inline tests/integration/helpers/reverse-test.js]: `s/does/does not`

File: addon/helpers/reverse.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  Helper,
+  observer,
+  A: emberArray,
+  isArray
+} = Ember;
+
+export default Helper.extend({
+  compute([array]) {
+    if (!isArray(array)) {
+      return [];
```
Comment (poteto) [inline addon/helpers/reverse.js]: Can we return `[array]`, and update our unit test? This might help with chaining.

**Comment by poteto** [comment]
LGTM apart from minor comments, thanks! :+1:

**Comment by spencer516** [comment]
@poteto Updates per your comments. :smiley:

File: addon/helpers/reverse.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+const {
+  set,
+  Helper,
+  observer,
+  A: emberArray,
```
Comment (martndemus) [inline addon/helpers/reverse.js]: Can you put `A: emberArray` up top? And also `Helper` on top.

```js
const {
  A: emberArray,
  Helper,
  observer,
  isArray,
  set
} = Ember;

**Comment by poteto** [comment]
If you rebase this against master, `beta` will be an allowed failure.

**Comment by martndemus** [comment]
@homu r+

**Pull request merged**